qt/Session.cc

```diff
@@ -545,6 +545,7 @@ std::vector<std::string_view> const& Session::getKeyNames(TorrentProperties prop
             TR_KEY_haveUnchecked,
             TR_KEY_haveValid,
             TR_KEY_isFinished,
+            TR_KEY_labels,
```

Comment:
Should this be moved to `MainInfoKeys`? These labels don't change unless a user initiates the change, so it seems wasteful to be constantly re-fetching them.

Reply:
done